webpack/.eslintrc.yml

```diff
@@ -0,0 +1,6 @@
+env:
+  browser: false
+  node: true
+
+rules:
+  import/no-extraneous-dependencies: [error, { devDependencies: true }]
```

Comment:
Does this file inherit rules from parent eslint config automatically?

Reply:
Yes. This is the idea of ESLint. All nested configs inherit and partially override rules.